youtube-helpers.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import json
+from youtube_dl import YoutubeDL
+
+
+def youtube_search(search_str):
+    """
+    Function takes in a search string and returns a dict object containing
+    Youtube search results.
+    {
+      "kind": "youtube#searchResult",
+      "etag": etag,
+      "id": {
+        "kind": string,
+        "videoId": string,
+        "channelId": string,
+        "playlistId": string
+      },
+      "snippet": {
+        "publishedAt": datetime,
+        "channelId": string,
+        "title": string,
+        "description": string,
+        "thumbnails": {
+          (key): {
+            "url": string,
+            "width": unsigned integer,
+            "height": unsigned integer
+          }
+        },
+        "channelTitle": string,
+        "liveBroadcastContent": string
+      }
+    """
+    api_key = "AIzaSyCfnPwKpsLH9lGp9bV5P3ZTI-rmZ3o8IPg"
```

Comment:
Sensitive information like API keys should be handled through environment variables or something of the sort. We'll need to remove this line from the code base and replace it with environment variables instead.

Generally speaking, we don't want to have API keys stored in version control or in the codebase. For the purposes of our project, it probably doesn't matter too much since it's basically just our group accessing this, but it's still best practice to handle them through environment variables. If an API key is publicly available, other people can use it for their own purposes and potentially abuse the API using our key (which could get our key revoked in some cases). 

We'll each need a copy of the environment variables locally, and we'll need to be able to store it on the server we host the web app on. Hope I explained everything properly, let me know if there's any questions on this.